docs/docs/cmd/spo/group/group-member-add.md

```diff
@@ -33,26 +36,75 @@ For the `--userName` or `--email` options you can specify multiple values by sep
 
 ## Examples
 
-Add a user with name _Alex.Wilber@contoso.com_ to the SharePoint group with id _5_ available on the web _https://contoso.sharepoint.com/sites/SiteA_
+Add a user with the userName parameter to a SharePoint group with the groupId parameter
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userName "Alex.Wilber@contoso.com"
 ```
 
-Add multiple users by name to the SharePoint group with id _5_ available on the web _https://contoso.sharepoint.com/sites/SiteA_
+Add multiple users with the userName parameter to a SharePoint group with the groupId parameter
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userName "Alex.Wilber@contoso.com, Adele.Vance@contoso.com"
 ```
 
-Add a user with email _Alex.Wilber@contoso.com_ to the SharePoint group with name _Contoso Site Owners_ available on the web _https://contoso.sharepoint.com/sites/SiteA_
+Add a user with the email parameter to a SharePoint group with the groupName parameter
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupName "Contoso Site Owners" --email "Alex.Wilber@contoso.com"
 ```
 
-Add multiple users by email to the SharePoint group with name _Contoso Site Owners_ available on the web _https://contoso.sharepoint.com/sites/SiteA_
+Add multiple users with the email parameter to a SharePoint group with the groupName parameter
 
 ```sh
 m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupName "Contoso Site Owners" --email "Alex.Wilber@contoso.com, Adele.Vance@contoso.com"
 ```
+
+Add a user with the userId parameter to a SharePoint group with the groupId parameter
+
+```sh
+m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userId 5
+```
+
+Add multiple users with the userId parameter to a SharePoint group with the groupId parameter
+
+```sh
+m365 spo group member add --webUrl https://contoso.sharepoint.com/sites/SiteA --groupId 5 --userId "5,12"
+```
+
+## Response
+
+=== "JSON"
+
+    ```json
+    [
+      {
+        "AllowedRoles": [
+          0
+        ],
+        "CurrentRole": 0,
+        "DisplayName": "John Doe",
+        "Email": "john.doe@contoso.onmicrosoft.com",
+        "InvitationLink": null,
+        "IsUserKnown": true,
+        "Message": null,
+        "Status": true,
+        "User": "i:0#.f|membership|john.doe@contoso.onmicrosoft.com"
+      }
+    ]
+    ```
+
+=== "Text"
+
+    ```text
+    DisplayName  Email
+    -----------  ---------------------------------
+    John Doe     john.doe@contoso.onmicrosoft.com
+    ```
+
+=== "CSV"
+
+    ```csv
+    DisplayName,Email
+    John Doe,john.doe@contoso.onmicrosoft.com
+    ```
```

Comment:
In visual studio code there is a white line at the end
![image](https://user-images.githubusercontent.com/35696168/208501788-15b26f3c-98dd-4539-879e-1565a2940202.png)

But I understand why you make this remark because i don't see that white line in the files changed tab here in github...